src/Node/TransNode.php

```diff
@@ -34,11 +34,37 @@
  */
 class TransNode extends Node
 {
-    /** @var string */
+    /**
+     * The label for gettext notes to be exported
+     *
+     * @var string
+     */
     protected static $notesLabel = '// notes: ';
 
-    public function __construct(Node $body, ?Node $plural, ?AbstractExpression $count, ?Node $notes, ?Node $domain = null, int $lineno = 0, ?string $tag = null)
-    {
+    /**
+     * Enable MoTranslator functions
+     *
+     * @var bool
+     */
+    protected static $enableMoTranslator = false;
```

Comment:
Is it not better to use `public` here? To avoid needing to extend the class to change that.

Reply:
I agree, and made the changes